Apps/Common/MultiPatchModelGenerator.C

```diff
@@ -0,0 +1,463 @@
+// $Id$
+//==============================================================================
+//!
+//! \file MultiPatchModelGenerator.C
+//!
+//! \date Sep 2 2016
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Multi-patch model generators for NURBS-based FEM simulators.
+//!
+//==============================================================================
+
+#include "MultiPatchModelGenerator.h"
+#include "ASMs2D.h"
+#include "ASMs3D.h"
+#include "IFEM.h"
+#include "SIMbase.h"
+#include "Utilities.h"
+#include "Vec3.h"
+#include "Vec3Oper.h"
+#include "tinyxml.h"
+
```

Comment:
to be able to compile locally I need an `#include "array"` is that an acceptable addition? Same in `ModelGenerator.C` btw